cmd/acceptance/main.go

```diff
@@ -141,6 +142,7 @@ func main() {
 		logger.Log("msg", "successfully destroyed cluster")
 
 	case run.FullCommand():
+		contextName = *runName
```

Comment:
For some reason, locally, `*runName` is empty when the test below starts running. Explicitly tracking this in a normal string variable fixes this, but it's nasty and I'd like to remove it if possible.